apis/opts/labels.go

```diff
@@ -22,6 +22,11 @@ func ParseLabels(labels []string) (map[string]string, error) {
 func parseLabel(label string) ([]string, error) {
 	fields := strings.SplitN(label, "=", 2)
 	if len(fields) != 2 {
+		// Only input key without value
+		if len(fields) == 1 {
+			fields = append(fields, "")
+			return fields, nil
+		}
 		return nil, fmt.Errorf("invalid label %s: label must be in format of key=value", label)
```

Comment:
If you compare the length of fields with 1, then this line of code is useless. Here is my reason.

Whatever the string label is, when it is been called with `fields := strings.SplitN(label, "=", 2)`, the length of fileds will only be 1 or 2. Then if `len(fields)` is not 2, then it must be 1. And as a result, code will never run to this line of code `return nil, fmt.Errorf("invalid label %s: label must be in format of key=value", label)`, and it is redudant.

How about doing this to simplify the code and make it more readable:

```
fields := strings.SplitN(label, "=", 2)
if len(fields) == 1 {
    fields = append(fields, "")
    return fields, nil
}
// length of fields must be 2 when code runs here
return fields, nil
```

In addition, we can judge whether to define this function `parseLabel ` to only return one parameter `[]string`, you see the `error` which is returned as the second parameter will always be nil. Thus no need to return it. But it depends on you about this part's modification.   @Flsorescence

Reply:
Thanks for your comment.

Yes, I know the statement `return nil, fmt.Errorf("invalid label %s: label must be in format of key=value", label)` will never be executed, but I don't know whether the second parameter is essential for future development.

Thus, I will annotate the useless statement and reserve the second parameter.

Reply:
Thanks for the update. I would like to invite @CodeJuan to take a review of this.